### ISSUE TITLE
In-App Command line interface to the bitcoin core through Tor

### ISSUE THREAD

**Issue by 1thales** [Fonta1n3/FullyNoded#251]
**Is your feature request related to a problem? Please describe.**
For bitcoin-cli functionalities that are not covered yet by FN. Using bash can give more flexibility

**Describe the solution you'd like**
It would be nice to have CLI through the mobile app without having to worry about tor ssh complexity in a general purpose Terminal

**Describe alternatives you've considered**
Ssh through Tor on Mac/iOS.

**Comment by Fonta1n3** [comment]
Yes I am planning on this. But it'll be a dedicated app. Need to finish a few things first.
